tests/exec-steps.sh

```diff
@@ -63,16 +63,8 @@ for STEP in /var/lib/tests/step-*.sh; do
     fi
 
     git config --global --unset init.templateDir
-    git config --global --unset githooks.shared
```

Comment:
Why did these all get deleted?

Reply:
Because its not needed. Since we call uninstall.sh before and that should do the reset, if anything. Maybe we could implement a test after the uninstall to make sure **no** githooks variables are set anymore.

Reply:
I implement a fail-as-early-as -possible test after the uninstall in `exec-steps.sh`: no githooks settings set ...
Check latest commits.